units/systemd-loop@.service

```diff
@@ -0,0 +1,22 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Attach File %f to Loopback Block Device
+Documentation=man:systemd-loop@.service(8)
+DefaultDependencies=no
+RequiresMountsFor=%f
+IgnoreOnIsolate=yes
```

Comment:
Is this work without udevd? Is it possible to use on containers?

Reply:
udev is not necessary. (we no longer watch udev events, we just do synchronous ioctls()).

we indeed do need to have an explicit dep on the loop.ko, will add. The DM one otoh is not necessary, we don#t do DM stuff here.

Reply:
And I think ConditionVirtualization=!container would be wrong: if this ends up in the boot process then we really should make things fail, but ConditionXYZ= is about skipping not about failing. 

Or in other words: if under some circumstances this logic is not desired, then it should not be the unit itself that is skipped but the thing that pulled it in in the first place.